server/app/lib/actions/fusor/activation_key/add_subscriptions.rb

```diff
@@ -36,18 +37,30 @@ def plan(activation_key_id, subscription_descriptions, repositories)
         def run
           ::User.current = ::User.find(input[:user_id])
           key = ::Katello::ActivationKey.find(input[:activation_key_id])
-          associate_subscriptions(key, input[:subscription_descriptions])
+          associate_subscriptions(key, input[:hostgroup])
           enable_repositories(key, input[:repository_cp_labels])
         ensure
           ::User.current = nil
         end
 
         private
 
-        def associate_subscriptions(key, subscription_descriptions)
-          subscription_descriptions.each do |description|
-            subscriptions = key.available_subscriptions.find_all { |sub| sub.description == description }
-            subscriptions.each { |subscription| key.subscribe(subscription.id, 0) } if subscriptions
+        def associate_subscriptions(key, hostgroup)
+          product_name = hostgroup[:activation_key][:content].parameterize.underscore.to_sym
```

Comment:
Why are we doing the parameterize? I'm trying this out in rails console and it doesn't seem to work.

```
2.2.4 :036 > product_name = hostgroup[:activation_key][:content].parameterize.underscore.to_sym
NoMethodError: undefined method `parameterize' for :rhevm:Symbol
        from (irb):36
        from /home/devel/jesusr/dev/rhci/.rvm/gems/ruby-2.2.4/gems/railties-4.1.14.2/lib/rails/commands/console.rb:90:in `start'
        from /home/devel/jesusr/dev/rhci/.rvm/gems/ruby-2.2.4/gems/railties-4.1.14.2/lib/rails/commands/console.rb:9:in `start'
        from /home/devel/jesusr/dev/rhci/.rvm/gems/ruby-2.2.4/gems/railties-4.1.14.2/lib/rails/commands/commands_tasks.rb:69:in `console'
        from /home/devel/jesusr/dev/rhci/.rvm/gems/ruby-2.2.4/gems/railties-4.1.14.2/lib/rails/commands/commands_tasks.rb:40:in `run_command!'
        from /home/devel/jesusr/dev/rhci/.rvm/gems/ruby-2.2.4/gems/railties-4.1.14.2/lib/rails/commands.rb:17:in `<top (required)>'
        from bin/rails:4:in `require'
        from bin/rails:4:in `<main>'
```